src/route-utils.js

```diff
@@ -139,11 +139,20 @@ class FilterState {
   constructor(state) {
     this.state = state;
     if (this.state.repoInfos.length === 0) {
-      this.state.repoInfos.push({
-        repoName: 'coala',
-        repoOwner: 'coala'
-      });
+      const repositories = process.env.REPOSITORIES || 'coala';
```

Comment:
you didnt read what I wrote last time.  there is no `env.REPOSITORIES` in the app, which is running in the client web-browser.

Reply:
Oh! I thought the webpack build is using this variable. Should I include it in the webpack config?